misc/deployments/test4.gno.land/genesis.json

```diff
@@ -15,6 +15,26 @@
       ]
     }
   },
+  "validators": [
+    {
+      "address": "g1mxguhd5zacar64txhfm0v7hhtph5wur5hx86vs",
+      "pub_key": {
+        "@type": "/tm.PubKeyEd25519",
+        "value": "C0l3PwIhnCmIG3YiAa9jf/uuvZj1ob7lolasnEhDgBE="
+      },
+      "power": "",
```

Comment:
@zivkovicmilos which is the voting power we are going to have? I might have missed it.

Reply:
We will all have the same voting power - `1` -- I'll update this in the main PR